src/devices/controller/analog-button.cpp

```diff
@@ -0,0 +1,42 @@
+#include "analog-button.h"
+
+#include <Arduino.h>
+
+AnalogButton::AnalogButton(int pin)
```

Comment:
Nit: For next time, please include restructuring refactors in another PR to reduce size.

Reply:
This was an artifact, I hadn't rebased this after submitting the other pull request.